### ISSUE TITLE
Ranker program updates

### ISSUE THREAD

**Issue by DeniseSl22** [DeniseSl22/VOCsArticle#2]
- Add Brenda database (http://www.brenda-enzymes.org/)
- Bio-assay-identifiers filter option per organism
- URLs included for further research

**Comment by jacobwindsor** [comment]
Taking a closer look at the duplicated bio-pathways, they are not *actually* duplicated. PubChem defines identical pathways in different organisms as separate entities, [see here](https://pubchem.ncbi.nlm.nih.gov/compound/10722#section=Biomolecular-Interactions-and-Pathways). So, a nice feature might be the ability to filter by organism. In PubChem, you could probably traverse each pathway, get the KEGG organism and filter by that

**Comment by DeniseSl22** [comment]
Yes that would be a nice addition! Since the data-set used to test the Ranker Program only contained human samples, this was not necessary before I am assuming?
However, I think adding URLs (as we discussed) could help potential users in the future tremendously...
I changed the issue according to your comments ;)

**Comment by DeniseSl22** [comment]
After viewing the code on https://github.com/jacobwindsor/pubchem-ranker, I was wondering if we can make the RP more suitable for other people to use. Not only by allowing the use of multiple type of files, but also by giving users one template that they have to fill in (e-mail for PubChem, data set name etc), while all the other files run on the background.
In this way, users do not have to search through the code, which makes the RP easier to apply.

**Comment by jacobwindsor** [comment]
@DeniseSl22 I've added issues to the PubChem ranker repo for this. [see here](https://github.com/jacobwindsor/pubchem-ranker/issues). Can you take a look/add any more you think are needed?

**Comment by DeniseSl22** [comment]
Done! Thanks for adding me ;)
